sources/logdog/src/main.rs

```diff
@@ -12,7 +12,12 @@ logs are at: /tmp/bottlerocket-logs.tar.gz
 
 # Logs
 
-For the commands used to gather logs, please see [log_request](src/log_request.rs).
+For the log requests used to gather logs, please see the following:
+
+* [log_requests](src/log_requests.rs)
```

Comment:
Looks like the last force-push fixed the README but not here where it's generated from?